components/script/clipboard_provider.rs

```diff
@@ -30,24 +29,3 @@ impl ClipboardProvider for ScriptToConstellationChan {
         .unwrap();
     }
 }
-
-pub struct DummyClipboardContext {
```

Comment:
It turns out that we actually need this for the tests in `tests/unit/script/textinput.rs`, so let's move it to that file instead.

Reply:
I've moved `DummyClipboardContext` to `test/unit/script/textinput.rs`.